src/modules/vendor/components/programs/ProfileContent.vue

```diff
@@ -4,7 +4,7 @@
       <p class="text-weight-bold">Sous-programme {{ index + 1 }}</p>
       <ni-input v-model.trim="program.subPrograms[index].name" required-field caption="Nom" @focus="saveTmpName(index)"
         @blur="updateSubProgramName(index)" :error="$v.program.subPrograms.$each[index].name.$error" />
-      <draggable @end="dropStep(subProgram._id)" v-model="subProgram.steps"
+      <draggable @end="dropStep(subProgram._id)" v-model="subProgram.steps" @start="saveSteps(subProgram._id)"
```

Comment:
Tu as essayé avec la propriété @change de draggable ?
'change event
change event is triggered when list prop is not null and the corresponding array is altered due to drag-and-drop operation'
https://github.com/SortableJS/Vue.Draggable

ça a l'air adapté à ce que tu veux faire et ça évite de rajouter plein de variables

Reply:
haha et en effet ...